src/composables/useCurrentUser.js

```diff
@@ -16,6 +16,53 @@ export function useCurrentUser() {
 		return pkp.currentUser.id;
 	}
 
+	function getCurrentUserName() {
+		return pkp.currentUser.username;
+	}
+
+	function getCurrentUserFullName() {
+		return pkp.currentUser.fullName;
+	}
+
+	function getCurrentUserInitials() {
+		return pkp.currentUser.initials;
+	}
+
+	function getUserLoggedInAsUserName() {
+		if (!isUserLoggedInAs()) {
+			return null;
+		}
+
+		const userLoggedInAs = getUserLoggedInAs();
+		return userLoggedInAs?.username || '';
+	}
+
+	function getUserLoggedInAsInitials() {
+		if (!isUserLoggedInAs()) {
+			return null;
+		}
+
+		const userLoggedInAs = getUserLoggedInAs();
+		return userLoggedInAs?.initials || '';
```

Comment:
This should cover the `null` return

```suggestion
		const userLoggedInAs = getUserLoggedInAs();
		return userLoggedInAs?.initials;
```

Alternatively (is the `if` is required for some reason...)


```suggestion
		if (!isUserLoggedInAs()) {
			return null;
		}

		const userLoggedInAs = getUserLoggedInAs();
		return userLoggedInAs.initials;
```